dags/kpi_forecasting.py

```diff
@@ -0,0 +1,93 @@
+"""
+See [kpi-forecasting in the docker-etl repository]
+(https://github.com/mozilla/docker-etl/blob/main/jobs/kpi-forecasting).
```

Comment:
Can we add some more context to these docs? Particularly, I'd like to know why it only runs once per week, what are the outputs of this DAG, and what is it powering dashboards that stakeholders see?